### DISCUSSION TITLE
Add history

### DISCUSSION THREAD

**Discussion by Rokcy98** [d4rken-org/sdmaid-se#361] [Ideas]
A section with cleaning history should be implemented, as it is on SD maid

**Comment by Helpful-Neko** [comment]
I think that's a good idea too

**Comment by d4rken** [comment]
What kind of information should be collected? This will also consume some space.

Can you elaborate when you would open the "history screen" and what information you would try to get from it?

**Comment by Helpful-Neko** [comment]
> What kind of information should be collected? This will also consume some space.
> 
> Can you elaborate when you would open the "history screen" and what information you would try to get from it?

I'm not sure what Rocky98 is wanting out of a history option. But I think he might be referring to this. 
![Screenshot_20230414-045805-565](https://user-images.githubusercontent.com/126541816/231996796-7f4be938-b75b-448c-9c60-43839bd93f78.png)
The Statistics feature in SD Maid 1 

I personally thought it was a pretty cool feature to see how much space you've saved (I'm a nerd and like nerdy things lol) but I can also see the possible benefit of not having it / being able to disable it. (For the reason you where saying d4rken about it taking up space)
